test/vanilla/requirements.txt

```diff
@@ -3,7 +3,7 @@ pytest
 pytest-cov
 pytest-asyncio==0.10.0;python_full_version>="3.5.2"
 async_generator;python_full_version>="3.5.2"
-msrest>=0.6.12
+git+https://github.com/Azure/msrest-for-python@xml_text
```

Comment:
won't merge this PR until the msrest has released with the change. Will change this back before merging